shard-operator-parent/shard-operator-core/src/main/java/com/redhat/service/smartevents/shard/operator/core/networking/KubernetesNetworkingService.java

```diff
@@ -41,8 +42,9 @@ public KubernetesNetworkingService(KubernetesClient client, TemplateProvider tem
     }
 
     @Override
-    public EventSource buildInformerEventSource(String operatorName, String component) {
-        return EventSourceFactory.buildIngressesInformer(client, operatorName, component);
+    public EventSource buildInformerEventSource(EventSourceContext<?> eventSourceContext, String operatorName, String component) {
+        // As the Ingress is targeting the istio-gateway and is not deployed in the same namespace of the CR we have to set the annotations with the primary resource references
```

Comment:
Is this going to change with the `rhose-gateway`?

Reply:
Nope